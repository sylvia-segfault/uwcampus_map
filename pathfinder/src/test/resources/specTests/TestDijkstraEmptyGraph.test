# Find a path with Dijkstra's algorithm on
# an empty graph

CreateGraph empty
FindPath empty A B