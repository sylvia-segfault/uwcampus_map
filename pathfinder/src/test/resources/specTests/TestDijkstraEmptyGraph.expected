# Find a path with Dijkstra's algorithm on
# an empty graph

created graph empty
unknown node A
unknown node B